### PR TITLE
Issue #39. Default pages content type and theming.

### PR DESCRIPTION
**Pull request by nhiieL**

### CODE REVIEW

**Comment by nhiieL** [comment]
sitepage

**Comment by nhiieL** [comment]
tabs-aligned

File: drupal/web/themes/custom/drupalcamp_cebu_2018/page.html.twig [hunk 1]
```diff
@@ -42,41 +42,45 @@
...
+      <header role="banner">
+        {{ page.header }}
+      </header>
 
-  {{ page.breadcrumb }}
+      {{ page.primary_menu }}
+      {{ page.secondary_menu }}
 
-  {{ page.highlighted }}
+      {{ page.breadcrumb }}
```
Comment (Luukyb) [inline drupal/web/themes/custom/drupalcamp_cebu_2018/page.html.twig]: breadcrumb should be inside the grid.

File: drupal/web/themes/custom/drupalcamp_cebu_2018/page.html.twig [hunk 2]
```diff
@@ -42,41 +42,45 @@
...
 
-  {{ page.help }}
+      {{ page.highlighted }}
```
Comment (Luukyb) [inline drupal/web/themes/custom/drupalcamp_cebu_2018/page.html.twig]: I think this should be inside the grid.

**Pull request merged**